api/envoy/api/v2/rds.proto

```diff
@@ -9,6 +9,7 @@ option java_package = "io.envoyproxy.envoy.api.v2";
 option java_generic_services = true;
 
 import "envoy/api/v2/core/base.proto";
+import "envoy/api/v2/core/config_source.proto";
```

Comment:
You need to fix the Bazel target deps to include config_source to fix the build.

Reply:
Fixed.